news/644.new.rst

```diff
@@ -0,0 +1 @@
+If `nltk_data` can be found both in the frozen program and under the default location specified by `NLTK`, the former should be preferred to the latter.
```

Comment:
Use two backticks to quote nltk_data and NLTK. The file should be called `646.update.rst`.